Rldx/Rldx/Managers/ResourceManager/DxResourceManager.cpp

```diff
@@ -147,28 +147,31 @@ namespace rldx
 		}*/
 	}
 
-	void DxResourceManager::GetResourcesFromCallBack(QList<QString>& qstrMissingFiles, QList<QByteArray>& destBinaries)
+	void DxResourceManager::GetResourcesFromCallBack(std::vector<std::wstring>* qstrMissingFiles, std::vector<std::vector<unsigned char>>* destBinaries)
 	{
 		if (!sm_assetCallBack) {
 			throw std::exception("No asset callback function set");
 		}
 
-		sm_assetCallBack(&qstrMissingFiles, &destBinaries);
+		sm_assetCallBack(qstrMissingFiles, destBinaries);
```

Comment:
And ofc change this to
`sm_assetCallBack(qstrMissingFiles, destBinaries);`

Reply:
Fixed in the last commit.